2025/webextensions-wg.html

```diff
@@ -159,285 +206,629 @@ <h2>Scope</h2>
         <p>The following things are in-scope for this Working Group:</p>
```

Comment:
```suggestion
        <p>The following features are in-scope for this Working Group:</p>
```

This terminology aligns with the template.